app/parse.py

```diff
@@ -8,9 +15,68 @@ class Quote:
     tags: list[str]
 
 
+def get_num_page() -> int:
+    num_page = 1
+
+    while True:
+        page = requests.get(urljoin(BASE_URL, f"page/{num_page}/")).content
```

Comment:
what if something doesn't go according to plan? 
I advise you to check whether you receive a status of 200 after the request